### DISCUSSION TITLE
PM2 / Supervisor Panel, & Port Forwarding Site

### DISCUSSION THREAD

**Discussion by kidino** [vitodeploy/vito#288] [Ideas]
I just learned about this and it's looking awesome. I saw your intro video too. Cool stuff.

I was wondering if you have plans for PM2 / Supervisor panel to help manage background services. I am thinking about running a websocket server, Reverb, or maybe NodeJS.

And to add to that, any plans for creating a port forwarding site? That way I can assign a domain to background service, with SSL.

**Comment by saeedvaziry** [comment]
After you create a site with VitoDeploy, there is a Queues section in the site details and you can manage supervisor there for your background services like reverb.

For port forwarding if you mean reverse proxy, Vito allows you to modify the nginx vhost configuration in the site settings which I believe you can make this happen.

But in the future we will add more site types to handle these out of the box

**Reply by kidino** [reply]
Cool. Thanks. I will check it out.
